waku/v2/protocol/waku_rln_relay/rln_relay.nim

```diff
@@ -0,0 +1,470 @@
+when (NimMajor, NimMinor) < (1, 4):
```

Comment:
Note: renamed utils.nim to rln_relay.nim

There is more refactoring to be done in terms of message validation, will create a tracking issue for it.